weldx/welding/groove/iso_9692_1.py

```diff
@@ -1575,19 +1585,19 @@ def _helperfunction(segment, array) -> geo.Shape:
         if elem == "arc":
             arr0 = [
                 # begin
-                array[0][counter],
+                array[0][counter].to(_DEFAULT_LEN_UNIT).magnitude,
                 # end
-                array[0][counter + 2],
+                array[0][counter + 2].to(_DEFAULT_LEN_UNIT).magnitude,
                 # circle center
-                array[0][counter + 1],
+                array[0][counter + 1].to(_DEFAULT_LEN_UNIT).magnitude,
             ]
             arr1 = [
                 # begin
-                array[1][counter],
+                array[1][counter].to(_DEFAULT_LEN_UNIT).magnitude,
                 # end
-                array[1][counter + 2],
+                array[1][counter + 2].to(_DEFAULT_LEN_UNIT).magnitude,
                 # circle center
-                array[1][counter + 1],
+                array[1][counter + 1].to(_DEFAULT_LEN_UNIT).magnitude,
             ]
```

Comment:
this should just be
```python
arr0 = array[0][[counter, counter + 2, counter +1]]
arr1 = array[1][[counter, counter + 2, counter +1]]
arr = np.vstack((arr0,arr1))
```
convert to magnitude after if necessary

when working with `pint` try to avoid list wherever possible, just stick to `numpy` style indexing and array operations